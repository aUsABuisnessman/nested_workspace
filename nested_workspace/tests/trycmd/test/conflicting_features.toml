args = ["test"]

[bin]
name = "cargo"

[fs]
cwd = "../../../../fixtures/conflicting_features"
