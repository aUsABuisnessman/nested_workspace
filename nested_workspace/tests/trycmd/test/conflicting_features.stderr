...
   Compiling conflicting_features v0.1.0 ([CWD])
    Finished `test` profile [unoptimized + debuginfo] target(s) in [..]s
     Running unittests src/lib.rs (target/debug/deps/conflicting_features-27179ee10fddcfd7)
<<< [CWD]/big_endian
   Compiling big_endian v0.1.0 ([CWD]/big_endian)
    Finished `test` profile [unoptimized + debuginfo] target(s) in [..]s
     Running unittests src/lib.rs (target/debug/deps/big_endian-e74e29070f5b4de7)
>>> [CWD]/big_endian
<<< [CWD]/little_endian
   Compiling little_endian v0.1.0 ([CWD]/little_endian)
    Finished `test` profile [unoptimized + debuginfo] target(s) in [..]s
     Running unittests src/lib.rs (target/debug/deps/little_endian-3ab4df1e31c86f7b)
>>> [CWD]/little_endian
<<< [CWD]/to_bytes
   Compiling to_bytes v0.1.0 ([CWD]/to_bytes)
    Finished `test` profile [unoptimized + debuginfo] target(s) in [..]s
     Running unittests src/lib.rs (target/debug/deps/to_bytes-adced94164a6a131)
>>> [CWD]/to_bytes
